schema.development.kf

```diff
@@ -91,6 +91,16 @@ table write_grants {
     foreign_key (wg_owner_user_id) references users(id) on_delete cascade
 }
 
+table consumed_write_grants {
```

Comment:
During the meeting, I'm pretty sure (but my memory sucks) that we settled on saving the whole message. So we either keep the message bytes, or we keep all the fields necessary to re-generate the message.

Reply:
You are correct in your exposition, but (again, if I recall correctly) keeping the whole message was decided to sidestep the problem of "what fields should we keep around". I proposed that we don't make a decision for now, keep everything, and trim things in the future when we're more familiar with how people will want to use the system.

I'll also disambiguate this with Martin.

Reply:
https://idosgroup.slack.com/archives/C07J0SDAW5U/p1738743317069499

Reply:
Added by this https://github.com/idos-network/idos-schema/pull/68/commits/3ec5a97ab291808b90dc0003c60fb5de40625cd3